recipes/ipyleaflet-gl-vector-layer-plugin/conda_build_config.yaml

```diff
@@ -0,0 +1,8 @@
+cudnn:                                    # [linux64]
+  - undefined                             # [linux64]
+cuda_compiler_version:                    # [linux64]
+  - None                                  # [linux64]
+docker_image:                             # [linux64]
+  - condaforge/linux-anvil-cos7-x86_64    # [linux64]
+cdt_name:                                 # [linux64]
+  - cos7                                  # [linux64]
```

Comment:
You probably do not need this file.

Reply:
The error is: `jupyter_packaging==0.12.0 is incompatible with jupyter_packaging~=0.7.9`. That is a bit odd, let me check a few things first.